.changeset/dirty-hornets-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@fuel-bridge/solidity-contracts': minor
+---
+
+add absolute path for the event logs in ci
```

Comment:
nit, the path does not look "relative" to me in the CI

Reply:
I meant absolute in my previous comment.

`path: ./packages/solidity-contracts/grantedRoles.json` is a relative path. Absolute paths look like `/path/to/file`,  in other words, they do not start with a `.`

What you did is fix the path as it was relative to the root of the project, whereas before it was being treated as relative to the solidity-contracts directory. The path is still relative :)